apps/jobbrowser/src/jobbrowser/apis/query_api.py

```diff
@@ -57,6 +58,19 @@ def _get_api(user, cluster=None):
     server_url = _get_impala_server_url(session)
   return get_impalad_api(user=user, url=server_url)
 
+def _convert_to_6_digit_ms_local_time(start_time):  
+  if '.' in start_time:
```

Comment:
@bjornalm @athithyaaselvam 
I am wondering if we should take care if there is no '.' in start_time.

Reply:
Added